game-store-service/tests/catalogBatchProcess.test.ts

```diff
@@ -0,0 +1,58 @@
+import catalogBatchProcess from "../functions/catalogBatchProcess";
+
+jest.mock("aws-sdk", () => ({
```

Comment:
Consider using the next package for mocking aws-sdk: https://www.npmjs.com/package/aws-sdk-mock

Reply:
Nice package. Definitely will try it.